static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -66,6 +88,12 @@ function SentryFunctionDetails(props: Props) {
     res.status(200).send(message);
   };`;
 
+  const [onIssue, setOnIssue] = useState(!!sentryFunction?.events?.includes('issue'));
+  const [onError, setOnError] = useState(!!sentryFunction?.events?.includes('error'));
+  const [onComment, setOnComment] = useState(
```

Comment:
@vpandiarajan20 I don't think we need three separate hooks for this, can't we just have one hook that stores an array of event types?